src/test/java/org/opentripplanner/model/plan/PlaceTest.java

```diff
@@ -50,7 +50,7 @@ public void acceptsNullCoordinates() {
         assertNull(p.coordinate);
     }
 
-    private static Stop stop(String stopId, double lat, double lon) {
+    public static Stop stop(String stopId, double lat, double lon) {
```

Comment:
There is the `Stop#stopForTest(String idAndName, double lat, double lon)` method witch could be used here. Even if it is not the best solution, it allow us to add fields to the stop class without having to edit this test.

Reply:
What would you think about moving the `stopForTest` method into a test helper?